app/grants/migrations/0011_auto_20190221_2050.py

```diff
@@ -22,5 +22,5 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
-            migrations.RunPython(forwards, backwards),
+            # migrations.RunPython(forwards, backwards),
```

Comment:
This step in the migrations always fails because of the `is_paid_plan` addition to the grants model.  Is there a way to ensure that migration #16 gets run before this one, given they are in the same app?